package.json

```diff
@@ -105,6 +105,7 @@
     "@cerner/terra-docs": "^1.0.0",
     "classnames": "^2.2.5",
     "eventemitter3": "^4.0.4",
+    "mutationobserver-shim": "0.3.3",
```

Comment:
Is there a reason why using 0.3.3 instead of the latest 0.3.7?

Reply:
to add to this, does anyone remember why we lock this down? is it because there still hasn't been a major release?